recipe/build.sh

```diff
@@ -1,16 +1,23 @@
 #!/usr/bin/env sh
+
+set -euxo pipefail
+
 # Get an updated config.sub and config.guess
 cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./build-aux
 cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./libcharset/build-aux
-set -ex
 
 ./configure --prefix=${PREFIX}  \
             --host=${HOST}      \
             --build=${BUILD}    \
             --enable-static     \
             --disable-rpath
 
-make -j${CPU_COUNT} ${VERBOSE_AT}
-if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
+if [[ "${target_platform}" == osx-* ]]; then
+    # make -f Makefile.devel CC="${CC}" CFLAGS="${CFLAGS}" totally-clean
+    make -f Makefile.devel CC="${CC}" CFLAGS="${CFLAGS}"
```

Comment:
```suggestion
    make -f Makefile.devel CC="${CC_FOR_BUILD}" CFLAGS="${CFLAGS}"
```

Reply:
I always, *always* forget about this one...